src/main/java/io/managed/services/test/client/securitymgmt/SecurityMgmtApi.java

```diff
@@ -1,60 +1,51 @@
+import com.openshift.cloud.api.kas.api.kafkas_mgmt.v1.V1RequestBuilder;
+import com.openshift.cloud.api.kas.api.kafkas_mgmt.v1.service_accounts.ServiceAccountListResponse;
 import com.openshift.cloud.api.kas.models.ServiceAccount;
-import com.openshift.cloud.api.kas.models.ServiceAccountList;
 import com.openshift.cloud.api.kas.models.ServiceAccountRequest;
 import io.managed.services.test.client.BaseApi;
 import io.managed.services.test.client.exception.ApiGenericException;
 import io.managed.services.test.client.exception.ApiUnknownException;
 
+import java.util.concurrent.TimeUnit;
+
 public class SecurityMgmtApi extends BaseApi {
 
     private final ApiClient apiClient;
-    private final SecurityApi api;
+    private final V1RequestBuilder v1;
 
     public SecurityMgmtApi(ApiClient apiClient, String offlineToken) {
         super(offlineToken);
         this.apiClient = apiClient;
-        this.api = new SecurityApi(apiClient);
-    }
-
-    @Override
-    protected ApiUnknownException toApiException(Exception e) {
-        if (e instanceof ApiException) {
-            var ex = (ApiException) e;
-            return new ApiUnknownException(ex.getMessage(), ex.getCode(), ex.getResponseHeaders(), ex.getResponseBody(), ex);
-        }
-        return null;
-    }
-
-    @Override
-    protected void setAccessToken(String accessToken) {
-        ((HttpBearerAuth) apiClient.getAuthentication("Bearer")).setBearerToken(accessToken);
+        this.v1 = apiClient.api().kafkas_mgmt().v1();
     }
 
     @SuppressWarnings("unused")
     public ServiceAccount getServiceAccountById(String id) throws ApiGenericException {
-        return retry(() -> api.getServiceAccountById(id));
+        return retry(() -> v1.service_accounts(id).get().get(10, TimeUnit.SECONDS));
     }
 
-    public ServiceAccountList getServiceAccounts() throws ApiGenericException {
-        return retry(() -> api.getServiceAccounts(null));
+    public ServiceAccountListResponse getServiceAccounts() throws ApiGenericException {
+        return retry(() -> v1.service_accounts().get().get(10, TimeUnit.SECONDS));
     }
 
     public ServiceAccount createServiceAccount(ServiceAccountRequest serviceAccountRequest) throws ApiGenericException {
-        return retry(() -> api.createServiceAccount(serviceAccountRequest));
+        return retry(() -> v1.service_accounts().post(serviceAccountRequest).get(10, TimeUnit.SECONDS));
     }
 
     public void deleteServiceAccountById(String id) throws ApiGenericException {
         // TODO: why does it return Error
-        retry(() -> api.deleteServiceAccountById(id));
+        retry(() -> v1.service_accounts(id).delete().get(10, TimeUnit.SECONDS));
     }
 
     public ServiceAccount resetServiceAccountCreds(String id) throws ApiGenericException {
-        return retry(() -> api.resetServiceAccountCreds(id));
+        return retry(() -> v1.service_accounts(id).reset_credentials().post().get(10, TimeUnit.SECONDS));
```

Comment:
The post here throws the error:
```
2023-03-14 18:36:16 INFO  [KafkaInstanceAPITest:169] create or retrieve service account 'mk-e2e-kaa-sa-my-username-if-local-aws'
2023-03-14 18:36:18 WARN  [SecurityMgmtAPIUtils:79] reset service account 'mk-e2e-kaa-sa-my-username-if-local-aws' credentials
2023-03-14 18:36:18 INFO  [BaseApi:39] java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
2023-03-14 18:36:18 ERROR [RetryUtils:154] io.managed.services.test.client.securitymgmt.SecurityMgmtApi.resetServiceAccountCreds(): skip error: 
java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
	at io.managed.services.test.client.BaseApi.handleException(BaseApi.java:47) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.handle(BaseApi.java:52) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.lambda$retry$0(BaseApi.java:56) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:133) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:119) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:95) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.retry(BaseApi.java:56) ~[classes/:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtApi.resetServiceAccountCreds(SecurityMgmtApi.java:44) ~[classes/:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtAPIUtils.applyServiceAccount(SecurityMgmtAPIUtils.java:80) ~[classes/:?]
	at io.managed.services.test.kafka.KafkaInstanceAPITest.testMaxMessageSizeLimit(KafkaInstanceAPITest.java:170) ~[test-classes/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.7.1.jar:7.7.1]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:829) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestRunner.run(TestRunner.java:602) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.run(SuiteRunner.java:330) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1176) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuites(TestNG.java:1099) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.run(TestNG.java:1067) ~[testng-7.7.1.jar:7.7.1]
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66) ~[testng-rt.jar:?]
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109) ~[testng-rt.jar:?]
Caused by: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096) ~[?:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtApi.lambda$resetServiceAccountCreds$4(SecurityMgmtApi.java:44) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.handleException(BaseApi.java:37) ~[classes/:?]
	... 36 more
Caused by: java.lang.IllegalArgumentException: method POST must have a request body.
	at okhttp3.Request$Builder.method(Request.kt:254) ~[okhttp-4.10.0.jar:?]
	at com.microsoft.kiota.http.OkHttpRequestAdapter.getRequestFromRequestInformation(OkHttpRequestAdapter.java:802) ~[microsoft-kiota-http-okHttp-0.2.0.jar:?]
	at com.microsoft.kiota.http.OkHttpRequestAdapter.lambda$getHttpResponseMessage$6(OkHttpRequestAdapter.java:647) ~[microsoft-kiota-http-okHttp-0.2.0.jar:?]
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run$$$capture(CompletableFuture.java:1773) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec$$$capture(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
2023-03-14 18:36:19 INFO  [BaseApi:39] java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
2023-03-14 18:36:19 ERROR [RetryUtils:154] io.managed.services.test.client.securitymgmt.SecurityMgmtApi.resetServiceAccountCreds(): skip error: 
java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
	at io.managed.services.test.client.BaseApi.handleException(BaseApi.java:47) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.handle(BaseApi.java:52) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.lambda$retry$0(BaseApi.java:56) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:133) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:145) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:119) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:95) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.retry(BaseApi.java:56) ~[classes/:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtApi.resetServiceAccountCreds(SecurityMgmtApi.java:44) ~[classes/:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtAPIUtils.applyServiceAccount(SecurityMgmtAPIUtils.java:80) ~[classes/:?]
	at io.managed.services.test.kafka.KafkaInstanceAPITest.testMaxMessageSizeLimit(KafkaInstanceAPITest.java:170) ~[test-classes/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.7.1.jar:7.7.1]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:829) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestRunner.run(TestRunner.java:602) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.run(SuiteRunner.java:330) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1176) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuites(TestNG.java:1099) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.run(TestNG.java:1067) ~[testng-7.7.1.jar:7.7.1]
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66) ~[testng-rt.jar:?]
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109) ~[testng-rt.jar:?]
Caused by: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096) ~[?:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtApi.lambda$resetServiceAccountCreds$4(SecurityMgmtApi.java:44) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.handleException(BaseApi.java:37) ~[classes/:?]
	... 37 more
Caused by: java.lang.IllegalArgumentException: method POST must have a request body.
	at okhttp3.Request$Builder.method(Request.kt:254) ~[okhttp-4.10.0.jar:?]
	at com.microsoft.kiota.http.OkHttpRequestAdapter.getRequestFromRequestInformation(OkHttpRequestAdapter.java:802) ~[microsoft-kiota-http-okHttp-0.2.0.jar:?]
	at com.microsoft.kiota.http.OkHttpRequestAdapter.lambda$getHttpResponseMessage$6(OkHttpRequestAdapter.java:647) ~[microsoft-kiota-http-okHttp-0.2.0.jar:?]
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run$$$capture(CompletableFuture.java:1773) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec$$$capture(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
2023-03-14 18:36:23 INFO  [BaseApi:39] java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
2023-03-14 18:36:23 ERROR [RetryUtils:154] io.managed.services.test.client.securitymgmt.SecurityMgmtApi.resetServiceAccountCreds(): skip error: 
java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
	at io.managed.services.test.client.BaseApi.handleException(BaseApi.java:47) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.handle(BaseApi.java:52) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.lambda$retry$0(BaseApi.java:56) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:133) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:145) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:145) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:119) ~[classes/:?]
	at io.managed.services.test.RetryUtils.retry(RetryUtils.java:95) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.retry(BaseApi.java:56) ~[classes/:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtApi.resetServiceAccountCreds(SecurityMgmtApi.java:44) ~[classes/:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtAPIUtils.applyServiceAccount(SecurityMgmtAPIUtils.java:80) ~[classes/:?]
	at io.managed.services.test.kafka.KafkaInstanceAPITest.testMaxMessageSizeLimit(KafkaInstanceAPITest.java:170) ~[test-classes/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.7.1.jar:7.7.1]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:829) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestRunner.run(TestRunner.java:602) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:437) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunner.run(SuiteRunner.java:330) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1176) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.runSuites(TestNG.java:1099) ~[testng-7.7.1.jar:7.7.1]
	at org.testng.TestNG.run(TestNG.java:1067) ~[testng-7.7.1.jar:7.7.1]
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66) ~[testng-rt.jar:?]
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109) ~[testng-rt.jar:?]
Caused by: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: method POST must have a request body.
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096) ~[?:?]
	at io.managed.services.test.client.securitymgmt.SecurityMgmtApi.lambda$resetServiceAccountCreds$4(SecurityMgmtApi.java:44) ~[classes/:?]
	at io.managed.services.test.client.BaseApi.handleException(BaseApi.java:37) ~[classes/:?]
	... 38 more
```
But the definition of post doesn't expect any argument.

Reply:
on it.

Reply:
A few things here, `SecurityMgmtApi` implementation is not correct, and we have found an issue with offline tokens that we will report.
Please @rkpattnaik780 hold off from the tests that are including `serviceAccounts` until I have a fix.

Reply:
Okay

Reply:
I fixed the used service account API on my branch, please cherry-pick or directly apply the changes from this commit: https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/pull/463/commits/78b7def11f45bb3a403bff4f9329bf3087c24cbd

Make sure you use an offline_token obtained through this page (as a temporary workaround):
https://console.redhat.com/openshift/token

Reply:
confirmed as resolved. ✅